OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessage.java

```diff
@@ -34,24 +43,42 @@
     @NonNull
     public ArrayList<ArrayList<OSTrigger>> triggers;
 
+    //Last IAM display time in seconds
+    private long lastDisplayTime = -1;
+    //Current quantity of displays
+    private int displaysQuantity = 0;
+    //Quantity of displays limit
+    private int displaysLimit = Integer.MAX_VALUE;
```

Comment:
Let's make display limit default to 1 since normal IAMs only show once?

Reply:
taking https://github.com/OneSignal/engineering/pull/92/files as reference, it says that if limit is missing make it MAX_INT

Reply:
@jkasten2 Did we decide to keep this `MAX_INT` as default?

Reply:
Yes, we can use `MAX_INT` incase the backend passes back a `redisplay` section but without an params. Maybe is just confusing as it is a default on the OSInAppMessage class instead of being defaulted in when parsing.

@Jeasmine @mikechoch What do you think about putting `displaysLimit` as `1` for this class default and using the `Integer.MAX_VALUE` if `redisplay` is present without any `limit` key in the parsing? Thinking this will make this more clear